backend/benefit/applications/services/pdf_templates/companies_benefit_with_de_minimis_aid.html

```diff
@@ -0,0 +1,24 @@
+{% block content %}
```

Comment:
Are the other templates missing `{% extends "base.html" %}` ?

Reply:
Nice catch, fixed them :)